query-engine/core/src/interactive_transactions/mod.rs

```diff
@@ -16,8 +16,7 @@ pub struct TxId(String);
 
 impl Default for TxId {
     fn default() -> Self {
-        #[allow(deprecated)]
-        Self(cuid::cuid().unwrap())
+        Self(cuid::cuid2())
```

Comment:
As discussed with @aqrln, we're now using `cuid2` for interactive transaction IDs (internal API).